meson.build

```diff
@@ -281,6 +281,10 @@ nnstreamer_install_conf.set('ENABLE_SYMBOLIC_LINK', get_option('enable-symbolic-
 nnstreamer_install_conf.set('TORCH_USE_GPU', get_option('enable-pytorch-use-gpu'))
 nnstreamer_install_conf.set('NNFW_RUNTIME_PRIORITIZE', get_option('nnfw-runtime-prioritize'))
 
+# Set framework priority about model file extension when automatically selecting framework for tensor filter.
+nnstreamer_install_conf.set('FRAMEWORK_PRIORITY_PB', get_option('framework-priority-pb'))
```

Comment:
plz move this block to line 101 (for both install-conf and test-conf)

nnstreamer_conf.set('FRAMEWORK_PRIORITY_PB', get_option('framework-priority-pb'))
nnstreamer_conf.set('FRAMEWORK_PRIORITY_TFLITE', get_option('framework-priority-tflite'))

Reply:
Updated.